### PR TITLE
Added FAQ entry for viewing logs

### PR DESCRIPTION
**Pull request by gniting**

### CODE REVIEW

**Review by NickM-27 — COMMENTED** [review]
This isn't totally accurate, frigate does store log files (otherwise you couldn't view logs inside the webui). I think this could be changed to `How to view Frigate logs without the webUI` and wording adjusted

**Comment by blakeblackshear** [comment]
You can also use a bind mount at `/dev/shm/logs` to persist the logs to disk if you want.

**Comment by gniting** [comment]
@NickM-27 @blakeblackshear Updated based on the feedback.

File: docs/docs/troubleshooting/faqs.md [hunk 1]
```diff
@@ -57,3 +57,35 @@ SQLite does not work well on a network share, if the `/media` folder is mapped t
 If MQTT isn't working in docker try using the IP of the device hosting the MQTT server instead of `localhost`, `127.0.0.1`, or `mosquitto.ix-mosquitto.svc.cluster.local`.
 
 This is because, by default, Frigate does not run in host mode so localhost points to the Frigate container and not the host device's network. 
+
+### How can I view  the Frigate log files without using the Web UI?
```
Comment (NickM-27) [inline docs/docs/troubleshooting/faqs.md:61]:
```suggestion
### How can I view the Frigate log files without using the Web UI?
```

File: docs/docs/troubleshooting/faqs.md [hunk 2]
```diff
@@ -57,3 +57,35 @@ SQLite does not work well on a network share, if the `/media` folder is mapped t
...
+Frigate uses Docker to manage logs. So to view these logs from the CLI, follow these steps:
+* Open a terminal or command prompt on the host running your Frigate container.
+* Type the following command and press Enter:
+  ```
+  docker logs -f frigate
+  ```
+  This command tells Docker to show you the logs from the Frigate container.
+  Note: If you've given your Frigate container a different name, replace "frigate" in the command with your container's actual name. The "-f" option means the logs will continue to update in real-time as new entries are added. To stop viewing the logs, press `Ctrl+C`. If you'd like to learn more about using Docker logs, including additional options and features, you can explore Docker's [official documentation](https://docs.docker.com/engine/reference/commandline/logs/).
+
+Alternatively, when you create the Frigate Docker container, you can bind a directory on the host to the mountpoint `/dev/shm/logs` to not only be able to persist the logs to disk, but also to be able to query them directly from the host using your fav log parsing/query util.
```
Comment (NickM-27) [inline docs/docs/troubleshooting/faqs.md:72]:
```suggestion
Alternatively, when you create the Frigate Docker container, you can bind a directory on the host to the mountpoint `/dev/shm/logs` to not only be able to persist the logs to disk, but also to be able to query them directly from the host using your fav log parsing/query util.

```

File: docs/docs/troubleshooting/faqs.md [hunk 3]
```diff
@@ -57,3 +57,35 @@ SQLite does not work well on a network share, if the `/media` folder is mapped t
 If MQTT isn't working in docker try using the IP of the device hosting the MQTT server instead of `localhost`, `127.0.0.1`, or `mosquitto.ix-mosquitto.svc.cluster.local`.
 
 This is because, by default, Frigate does not run in host mode so localhost points to the Frigate container and not the host device's network. 
+
+### How can I view  the Frigate log files without using the Web UI?
+
+Frigate uses Docker to manage logs. So to view these logs from the CLI, follow these steps:
```
Comment (NickM-27) [inline docs/docs/troubleshooting/faqs.md:63]:
```suggestion
Frigate manages logs internally as well as outputs directly to Docker via standard output. To view these logs using the CLI, follow these steps:
```

File: docs/docs/troubleshooting/faqs.md [hunk 4]
```diff
@@ -57,3 +57,36 @@ SQLite does not work well on a network share, if the `/media` folder is mapped t
...
+Frigate manages logs internally as well as outputs directly to Docker via standard output. To view these logs using the CLI, follow these steps:
+* Open a terminal or command prompt on the host running your Frigate container.
+* Type the following command and press Enter:
+  ```
+  docker logs -f frigate
+  ```
+  This command tells Docker to show you the logs from the Frigate container.
+  Note: If you've given your Frigate container a different name, replace "frigate" in the command with your container's actual name. The "-f" option means the logs will continue to update in real-time as new entries are added. To stop viewing the logs, press `Ctrl+C`. If you'd like to learn more about using Docker logs, including additional options and features, you can explore Docker's [official documentation](https://docs.docker.com/engine/reference/commandline/logs/).
+
+Alternatively, when you create the Frigate Docker container, you can bind a directory on the host to the mountpoint `/dev/shm/logs` to not only be able to persist the logs to disk, but also to be able to query them directly from the host using your fav log parsing/query util.
```
Comment (hawkeye217) [inline docs/docs/troubleshooting/faqs.md:72]:
```suggestion
Alternatively, when you create the Frigate Docker container, you can bind a directory on the host to the mountpoint `/dev/shm/logs` to not only be able to persist the logs to disk, but also to be able to query them directly from the host using your favorite log parsing/query utility.
```

**Review by hawkeye217 — CHANGES_REQUESTED** [review]

**Review by hawkeye217 — APPROVED** [review]

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**